static/app/components/lazyLoad.tsx

```diff
@@ -58,7 +45,7 @@ function LazyLoad<C extends ComponentType>(props: Props<C>) {
     [setError]
   );
 
-  const importComponent = props.component ?? props.route?.componentPromise;
+  const importComponent = props.component;
```

Comment:
Should you keep the old behavior as a way to safely transition getsentry or are you going to try and merge both changes and take one failing build?

Reply:
I'll merge both together and take the one failing build.

This should be good to go.

Reply:
https://github.com/getsentry/getsentry/pull/7556 is already green so I think we should be goof